src/main/python/ttconv/vtt/cue.py

```diff
@@ -42,6 +42,12 @@ class LineAlignment(Enum):
     center = "center"
     end = "end"
 
+  class TextAlignment(Enum):
+    """WebVTT text alignment cue setting"""
```

Comment:
That should be  "start", "center", "end", "left", "right"

https://www.w3.org/TR/webvtt1/#webvtt-alignment-cue-setting

Reply:
Okay, "middle" does not exist... I took a production sample as reference, I should not have!
In the updated PR, I kept the "left/right" notation because it seems more commonpractice with vtt (which inherited from srt with similar "pad" \anX tricks), despite modern TTML "prefer" start/end, I feel.
BTW, writing the test code, I realized that current TTML reader code parse "left/right" to "start/end" without setting writing mode which would be an issue for rl; but I think the TTML case "left/right" with right-left reading mode does not match many real world use cases.